File: src/main/java/br/ufc/insightlab/vonqbe/model/ReturnVirtuosoUpload.java
package br.ufc.insightlab.vonqbe.model;

public class ReturnVirtuosoUpload {

    public String name;

    public String linkURL;

    public String baseURI;

    public UploadFileResponse squema;

    public ReturnVirtuosoUpload() {
    }

    public ReturnVirtuosoUpload(String name, String linkURL, String baseURI, UploadFileResponse squema) {
        this.name = name;
        this.linkURL = linkURL;
        this.baseURI = baseURI;
        this.squema = squema;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLinkURL() {
        return linkURL;
    }

    public void setLinkURL(String linkURL) {
        this.linkURL = linkURL;
    }

    public String getBaseURI() {
        return baseURI;
    }

    public void setBaseURI(String baseURI) {
        this.baseURI = baseURI;
    }

    public UploadFileResponse getSquema() {
        return squema;
    }

    public void setSquema(UploadFileResponse squema) {
        this.squema = squema;
    }
}

File: app/scala/br/ufc/insightlab/vonqbe/model/ReturnVirtuosoUpload.scala
package scala.br.ufc.insightlab.vonqbe.model

import play.api.libs.json.{Format, Json}

class ReturnVirtuosoUpload (var name:String,var baseUri : String, var graphURI : String, var squema : UploadFileResponse){

  override def toString: String = "Virtuoso -------------\n" +
    "name : "+ name + "\n"+
    "baseURI : " + baseUri +"\n"+
    "graphURI : " + graphURI +"\n"+
    "squema : " + squema

  implicit val format : Format[ReturnVirtuosoUpload] = Json.format[ReturnVirtuosoUpload]
}